### ISSUE TITLE
[BUG]: Broken image thumbnail is not displayed in the editor

### ISSUE THREAD

**Issue by Maks-Yaremenko** [sibiraj-s/ngx-editor#309]
# I'm submitting a

- Bug report

## Description

You can successfully add an image using the add image tool, then you have to delete the image on the server. After you'll load a document that contained a deleted (on the server) image, the broken image won't be reflected in the document - but it will be there.

### Version Information

```
ngx-editor: 5.2.0
angular: 9.1.9
```

### Browser

- Chrome - Version 87.0.4280.88 (Official Build) (x86_64)

**Comment by sibiraj-s** [comment]
@Maks-Yaremenko Please follow the issue template.

Issue title and issue description are totally different.

**Comment by Maks-Yaremenko** [comment]
okay. sorry

**Comment by sibiraj-s** [comment]
based on my understanding I updated the title. can you confirm if that is the bug?

**Comment by sibiraj-s** [comment]
Can you paste the document sample with which you are facing the issue

**Comment by Maks-Yaremenko** [comment]
no no, incorrect. Let me explain - You can successfully add an image using the add image tool, then you have to delete the image on the server. After you'll load a document that contained a deleted (on the server) image broken image won't be reflected in the document - but it will be there

**Comment by sibiraj-s** [comment]
Oh. got it. the broken thumbnail is not showing up.

**Comment by Maks-Yaremenko** [comment]
yes, exactly!

**Comment by Maks-Yaremenko** [comment]
ps: is there a way to pass custom validator for image link? I am facing issues with some URLs, that contain сyrillic fonts, or some characters... like times.. for example when an image has dimensions like 1200x1500

**Comment by sibiraj-s** [comment]
Can you open a new discussion for that? lets keep the discussion here for this issue alone

**Comment by Maks-Yaremenko** [comment]
Do you mean request?

**Comment by sibiraj-s** [comment]
do here https://github.com/sibiraj-s/ngx-editor/discussions

**Comment by sibiraj-s** [comment]
<img width="404" alt="Screenshot 2021-01-08 at 3 56 07 PM" src="https://user-images.githubusercontent.com/20282546/104004719-15727b80-51ca-11eb-91ff-a16cc10afa15.png">

@Maks-Yaremenko I was able to see the broken thumbnail when I tested it.

**Comment by sibiraj-s** [comment]
@Maks-Yaremenko  Let me know if this is an issue or can be closed

**Comment by Maks-Yaremenko** [comment]
I can confirm that the thumbnail exists in the record of the document, but not reflecting in the editor.

in editor (image exist on the server)
![image](https://user-images.githubusercontent.com/22315741/104169974-10e7d600-5409-11eb-97e7-2bbfa36b8f1e.png)

in HTML (image exist on the server)
![image](https://user-images.githubusercontent.com/22315741/104170051-2eb53b00-5409-11eb-9ba7-6a03800d4856.png)

in the editor (the image was removed from the server, no placeholder in the editor)
![image](https://user-images.githubusercontent.com/22315741/104170102-455b9200-5409-11eb-9539-225de9dfb03c.png)

in the HTML (this image was removed from the server, placeholder shows correct thumbnail)
![image](https://user-images.githubusercontent.com/22315741/104170187-64f2ba80-5409-11eb-86cf-0a981c87efa1.png)

**Comment by sibiraj-sr** [comment]
>ps: sad to me that you making all fixes for the latest version... 

Migrating to v6 is very straightforward. Can you refer this one to upgrade https://sibiraj-s.github.io/ngx-editor/#/migration-5-6 let me know if you face any issues. I will assist you along the migration.

**Comment by sibiraj-s** [comment]
>ps: sad to me that you making all fixes for the latest version... 

Migrating to v6 is very straightforward. Can you refer this one to upgrade https://sibiraj-s.github.io/ngx-editor/#/migration-5-6 let me know if you face any issues. I will assist you along the migration.

**Comment by sibiraj-s** [comment]
v6 allows to extend features very easy both internally and for developers too. I heavily recommend you to upgrade it.

**Comment by Maks-Yaremenko** [comment]
yes, I used this guide to migrate to a newer version but received errors with childCount, same as on v 5.3. + I created a wrapper to use editor inside mat-form-field, so I got some style issues during migration. Unfortunately, I'm not allowed to spend any more time with the editor on my project. Probably I'll try to migrate one more time after work hours.

**Comment by sibiraj-s** [comment]
Sure.

**Comment by sibiraj-s** [comment]
Thanks for sharing the screenshots I was able to reproduce that. The Images without alt text is the ones that doesn't have thumbnails. The rest works fine. I will debug this.

**Comment by Maks-Yaremenko** [comment]
Thank you! You are doing a great job!

**Comment by sibiraj-s** [comment]
Thanks. Consider a star for the project. If you like the work.

**Comment by Maks-Yaremenko** [comment]
done)

**Comment by sibiraj-s** [comment]
Turns out it is just a css issue. Fixed in [v6.1.0-beta.1](https://github.com/sibiraj-s/ngx-editor/releases/tag/v6.1.0-beta.1).

Since you have v5. I suggest a small patch.

```css
.NgxEditor img {
  height: 100%;
}
```

**Comment by Maks-Yaremenko** [comment]
thank you a lot!
